packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -202,11 +469,43 @@ class GrpcGatewayRouteSpec extends GatewayRouteSpec {
               virtualServiceName: this.routeTarget.virtualServiceName,
             },
           },
+          rewrite: this.defaultHostname
+            ? {
+              hostname: {
+                defaultTargetHostname: this.defaultHostname ? 'ENABLED' : 'DISABLED',
+              },
+            }: undefined,
         },
         match: {
           serviceName: this.match.serviceName,
+          hostname: this.match.hostname?.bind(scope).hostnameMatch,
+          metadata: this.match.metadata?.map(metadata => metadata.bind(scope).headerMatch),
         },
       },
     };
   }
 }
+
+function renderHttpGatewayRouteRewrite(scope: Construct, defaultTargetHostname?: boolean, pathRewrite?: HttpGatewayRouteRewrite)
```

Comment:
This function is used exactly once... maybe inline it?

Reply:
Got it.